app/models/ability.rb

```diff
@@ -10,6 +10,10 @@ def initialize(user)
       can [:update], Card do |card|
         card.owned_by?(user)
       end
+      can [:read], User
+      can [:update], User do |userr|
+        userr == user
```

Comment:
Its not about cards. 
That only means that a user can only update his/her own profile.